### ISSUE TITLE
Bring back "Bulk change" feature as it was released in version 3.6

### ISSUE THREAD

**Issue by Viator76** [AdguardTeam/AdguardForAndroid#5270]
### Issue Details

1) Massive change, where it was possible to disable/enable individual modules, both for applications with browsers and individually, now there is none of this and it requires manually enabling/disabling EVERY application.
2) All applications separately, and combining them into subgroups is not necessary because EACH application from the subgroup must be configured separately,
Please return the features of 3.6 to 4.3

Оригинал,
1)Массовое изменение, позволяло отключать определенные модули или всю защиту программы для приложений с браузерами, только приложений или браузеров, в 4.3 можно отключить/включить Защиту и Фильтры тобишь включать/отключать модули нужно в КАЖДОМ приложение отдельно.
2) Все приложения были по отдельности, и объединять их в подгруппы не нужно, т.к. КАЖДОЕ приложение из подгруппы нужно настраивать отдельно и подобное объединение мешает это делать.

Пожалуйста верните эти функции из 3.6 в 4.3.

### Proposed solution

Featured

**Comment by maxikuzmin** [comment]
@Viator76 I didn't quite understand points 1 and 3. 
Could you please clarify what you mean by "disable/enable individual modules" in point 1, and what you mean by "subgroups" in point 3?

**Comment by Viator76** [comment]
By “disable/enable individual modules” I mean program functions from the Mass Change menu, namely AdGuard Protection, Filtering, HTTPS and something else in the enable/disable list.
Оригинал,
Под "отключать/включать отдельные модули" я понимаю функции программы из меню Массового Изменения, а именно Защита AdGuard , Фильтрование, HTTPS и что-то еще в списке на включение/отключение.

Subgroups are those items in the list of applications in which there are several applications, at the top of the list of applications, for example, Google Servise & Google Framework, why combine them if it requires separate settings for each application?
Оригинал,
Субгруппы это те пункты в списке приложений в которых несколько приложений, вверху списка приложений, например Google Servise & Google Framework еще что-то, зачем объединять, если это требует отдельной настройки для каждого приложения?

**Comment by maxikuzmin** [comment]
@muchqs yes, we did rework the Access WiFi/mobile data logic for the app. This was done for usability, as not all users need to disable WiFi/mobile data access completely.

**Comment by maxikuzmin** [comment]
@Viator76 
1. Насколько я знаю, большинство юзеров не используют рут доступ, поэтому для большинства юзеров массовое изменение в виде "Фильтровать только браузеры" или "Фильтровать только приложения" не несет смысла, т.к. без рут доступа в любом случае будут фильтроваться только браузеры
2. Для понимания какое приложение работает через AdGuard вы можете перейти в Настройки -> Фильтрация -> Сеть -> HTTPS-фильтрация -> Приложения с HTTPS-фильтрацией. 
Оттуда также удобно проставить нужные вам тогглы напротив каждого приложения и включить или отключить фильтрацию
3. Как было в версии 3.6 – неправильно. Никогда не было возможности делать настройку для определенного приложения, это был баг, поэтому в версии 4. мы доработали это и объединили в группы часть приложений

**Comment by maxikuzmin** [comment]
@Viator76 тем не менее предлагаю оставить в  ишью на голосование тему функции "Массовое изменение", мы отслеживаем самые залайканные "Запросы функций" и берем в работу.

По поводу иконок рядом с приложениями лучше открыть второй "Запрос функции", чтобы юзеры также принимали участие в голосовании по отдельной теме, так мы будем лучше понимать запросы по каждой функции

**Comment by Viator76** [comment]
> По поводу иконок рядом с приложениями лучше открыть второй "Запрос функции", чтобы юзеры также принимали участие в голосовании по отдельной теме, так мы будем лучше понимать запросы по каждой функции

Хорошо, разобью.

**Comment by Viator76** [comment]
Разбил предложение фич на две, тут Массовые Изменения и Группировка приложений, в 5286 об Иконках.

**Comment by maxikuzmin** [comment]
@Viator76 спасибо!

>Работа через AdGuard это не всегда с HTTPS, вам ли это не знать.

Безусловно вы правы, я написал о HTTPS фильтрации, потому что вы упомянули её изначально

>Никогда? Но оно работало, а значит не баг, а фича.., а вы убрали фичу и теперь, вместо захода в приложение из списка ты вынужден постоянно разворачивать группу, негодится.

Работало, визуально, но на деле это было не так. Некоторые приложения разбили по группам, т.к. у них одинаковый UID. На уровне фильтрации трафика мы видим только UID приложения, от которого идёт запрос, т.е в любом случае отключая в теории одно приложение из группы, мы отключаем все приложения из группы, т.к. у них одинаковый UID

**Comment by Viator76** [comment]
> На уровне фильтрации трафика мы видим только UID приложения, от которого идёт запрос, т.е в любом случае отключая в теории одно приложение из группы, мы отключаем все приложения из группы, т.к. у них одинаковый UID

1) Т.е. группа названная Системные приложения тоже имеет один UID? Очень интересно, особенно когда смотришь на часть таких приложений и видишь в настройках отличия то HTTPS отключено в одних, то просто Фильтрация, как это соотносится с ОДНИМ UID? 
2)Тогда дополнение, пусть группы будут, но дать возможность держать группу развернутой ВСЕГДА, ну и иконки, но это отдельно.

**Comment by Viator76** [comment]
Добавлю, сорри не углядел, во всех таких приложениях из групп где то нет HTTPS, то еще чего отключено индивидуально, имеется одно совпадение, отключена Защита т.е. все остальное значения не имеет.
